x-pack/legacy/plugins/monitoring/public/components/elasticsearch/shard_allocation/_shard_allocation.scss

```diff
@@ -14,51 +14,55 @@ monitoring-shard-allocation {
   cluster-view {
     display: block;
   }
-  .parent {
-    padding-top: 14px;
-    border-left: 3px solid $euiColorSuccess !important;
-    &.red {
-      border-left: 3px solid $euiColorDanger !important;
-    }
-    &.yellow {
-      border-left: 3px solid $euiColorWarning !important;
-    }
-  }
-  td.unassigned {
+
+  .unassigned {
```

Comment:
For this file, if you can, I would highlight suggest adding more specific selectors with the `.mon` for each of the elements.